src/Umbraco.Web.UI.Client/src/less/components/umb-layout-selector.less

```diff
@@ -51,7 +51,9 @@
    border: 1px solid @blue;
 }
 
-.umb-layout-selector__dropdown-item-icon {
+.umb-layout-selector__dropdown-item-icon,
+.umb-layout-selector__dropdown-item-icon[class^="icon-"],
+.umb-layout-selector__dropdown-item-icon[class*=" icon-"] {
```

Comment:
Ensure it also works in listview layout selector.

**Before**
![image](https://cloud.githubusercontent.com/assets/2919859/14226113/1d379a8e-f8d9-11e5-95e3-406d287eca10.png)

**After**
![image](https://cloud.githubusercontent.com/assets/2919859/14226119/32f022ce-f8d9-11e5-88f3-c3ec428a58e3.png)